client/my-sites/plans/current-plan/jetpack-product-install/index.tsx

```diff
@@ -287,6 +288,20 @@ export class JetpackProductInstall extends Component< Props, State > {
 				status_akismet: status ? status.akismet_status : '(unknown)',
 				status_vaultpress: status ? status.vaultpress_status : '(unknown)',
 			} );
+
+			this.props.logToLogstash( {
+				feature: 'calypso_client',
+				message: 'Jetpack plugin installer error',
+				...( this.props.siteId && { site_id: this.props.siteId } ),
```

Comment:
I just realised we don't have `siteId` recorder in the above error event. Totally no need to add in this PR, just noting down.